src/Controllers/bookscontroller.php

```diff
@@ -1,11 +1,7 @@
 <?php
```

Comment:
Notice that I've renamed the `controller.php` file to `bookscontroller.php`. The filename being different from the classname is the reason we could not autoload the `BooksController` into the `Router`! When preparing autoload files, composer will look for classes based on filename. Since the file was named `controller.php`, but the class is `BooksController`, they don't match and the file cannot be loaded into other classes.